solarforecastarbiter/reports/figures.py

```diff
@@ -313,7 +336,7 @@ def scatter(timeseries_value_cds, timeseries_meta_cds, units):
     return fig
 
 
-def construct_metrics_cds(metrics, rename=None):
+def construct_metrics_dataframe(metrics, rename=None):
```

Comment:
Instead of deleting the existing code, we could create `figures/plotly.py` and `figures/bokeh.py`. Maybe not really needed but multiple plotting "backends" could be useful at some point.

Reply:
I think that's a really good idea. We could add logic to select the processing function based on the sfa reports version. Might require some minor tweaking to the bokeh handling, but I think that'd be a good idea.